CHANGELOG.D/2903.feature

```diff
@@ -0,0 +1,4 @@
+Added support of cluster energy schedules.
+`neuro config show --energy` will display awailable energy schedule timeslots.
```

Comment:
```suggestion
`neuro config show --energy` will display awailable energy schedule periods.
```

Reply:
done in https://github.com/neuro-inc/neuro-cli/pull/2907, thx!